lib/SPIRVProducerPass.cpp

```diff
@@ -43,7 +43,7 @@
 #include "llvm/Support/raw_ostream.h"
 #include "llvm/Transforms/Utils/Cloning.h"
 
-#include "spirv/1.0/spirv.hpp"
+#include "spirv/unified1/spirv.hpp"
```

Comment:
I'm a little leary about this change without making all of clspv use the unified headers. If it's not required, I'd prefer doing this separately (a la #421, which should really get finished).

Reply:
Yep, that is required for my next PR adding sub_groups support. I'll stash it till then.

Reply:
Will #421 be pulled in soon?

Reply:
I will try to get it finished  by early next week.